docs/en/query_language/functions/url_functions.md

```diff
@@ -12,15 +12,84 @@ Returns the protocol. Examples: http, ftp, mailto, magnet...
 
 ### domain
 
-Gets the domain. Cut scheme with size less than 16 bytes.
+Extracts the host part from URL.
+
+```
+domain(url)
+```
+
+**Parameters**
+
+- `url` — URL. Type: [String](../../data_types/string.md).
+
+
+URL can be specified with or without scheme. Examples:
```

Comment:
why examples are not in examples section?

Reply:
Because that is not the examples of function usage, that's the examples of arguments.